python/src/QuantileMatchingFactory_doc.i.in

```diff
@@ -0,0 +1,177 @@
+%feature("docstring") OT::QuantileMatchingFactory
+"Estimation by matching quantiles.
+
+Implements generic estimation by matching quantiles.
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    The distribution defining the parametric model to be adjusted to data.
+    Its parameters define the starting point of the algorithm.
+alpha : sequence of float, optional
+    The quantile levels :math:`\alpha_i`
+bounds : :class:`~openturns.Interval`, optional
+    Parameter bounds
```

Comment:
This is an optional parameter, so that its default value must be defined.
```suggestion
    Parameter bounds. The default bounds is an empty interval, which implies that the optimization problem is unbounded.
```